hazelcast-sql/src/test/java/com/hazelcast/sql/impl/plan/cache/PlanCacheTestSupport.java

```diff
@@ -52,6 +55,11 @@
         PART_MAP_2 = Collections.singletonMap(memberId, partitions2);
     }
 
+    @After
+    public void after() {
+        getPlanCache(instance()).getPlans().clear();
```

Comment:
Isn't similar logic already part of `SqlTestSupport`?

Reply:
Good catch! Thanks.